### DISCUSSION TITLE
How to set multiple values / how to control blinds with HmIP-FBL?

### DISCUSSION THREAD

**Discussion by jschroeter** [rdmtc/RedMatic#535] [Q&A] [answered]
Hello,

first of all: thanks for this awesome project, I'm very happy with RedMatic!

I'm trying to automate blinds and want to set the height and the slat position of HmIP-FBL actors. But I'm still having issues, most importantly that the blinds open 100% but then shortly after will move down a bit again. According to some forum posts it's important to always set both values, first the wanted slat position (LEVEL_2) and then the height (LEVEL). How can I do that with the CCU nodes? My current solution is the following but it still doesn't work reliably and I'm wondering if it's related to my flow:
<img width="408" alt="ccu-value" src="https://user-images.githubusercontent.com/487046/152935484-e31ea83e-0949-431f-b694-c7d6b0aa91dd.png">

How can I make sure that first the LEVEL_2 is set successfully and then LEVEL is set?

Thanks!
Jakob

**Comment by jschroeter** [comment, marked as answer]
I found a solution, Homematic provides a way to do it (COMBINED_PARAMETER datapoint): https://github.com/rdmtc/node-red-contrib-ccu/issues/154#issuecomment-1079763871

**Comment by hobbyquaker** [comment]
Beantwortet (COMBINED_PARAMETER, siehe Link im Thread). Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
